grapple/helpers.py

```diff
@@ -17,22 +17,28 @@
 field_middlewares = {}
 
 
-def register_streamfield_block(cls):
-    base_block = None
-    for block_class in inspect.getmro(cls):
-        if block_class in registry.streamfield_blocks:
-            base_block = registry.streamfield_blocks[block_class]
-
-    streamfield_types.append(
-        {
-            "cls": cls,
-            "type_prefix": "",
-            "interface": StreamFieldInterface,
-            "base_type": base_block,
-        }
-    )
-
-    return cls
+def register_streamfield_block(cls=None, extra_interfaces=()):
+    def decorator(cls):
```

Comment:
also, let's change `cls` to `klass` as it mirrors `cls` from parent function

Reply:
I've changed the name in the outer decorator to keep the inner decorator variable consistent with the dict key.